src/Symfony/Component/Security/Guard/GuardAuthenticatorInterface.php

```diff
@@ -39,11 +39,16 @@
      * Whatever value you return here will be passed to getUser() and checkCredentials()
      *
      * For example, for a form login, you might:
-     *
-     *      return array(
-     *          'username' => $request->request->get('_username'),
-     *          'password' => $request->request->get('_password'),
-     *      );
+     * 
+     *      if($request->request->has('_username')) {
```

Comment:
CS needs a fix: space after `if`, and `else` on the same line as `}` (but is the `else` required?)